docs/intro.md

```diff
@@ -0,0 +1,21 @@
+---
+id: intro
+title: Amphora HTML
+sidebar_label: Amphora HTML
+---
+
+---
+The HTML renderer for Clay components that use [Handlebars](http://handlebarsjs.com/) templates.
+
+---
+## Why?
+
+HTML rendering was controlled entirely by [Amphora](https://github.com/nymag/amphora) in v2.x, which meant that Amphora was responsible for a lot of heavy lifting. Furthermore, rendering was handled by a module called [Multiplex Templates](https://www.npmjs.com/package/multiplex-templates) which did some :crystal_ball: dark magic :sparkles: to synchronously render component templates. This required some affordances from Amphora which added to the weight of the middleware. By separating renderers out into separate modules which can be plugged into Amphora >v2.x we can create renderers for specific requirements (XML, Amp, etc.) _and_ the rendering process can be moved to separate machines to create more maintainable systems.
```

Comment:
We could add some style to `>v2.x` maybe bold to highlight this important info

Reply:
Done!! Thanks!!